css/index.css

```diff
@@ -167,31 +173,32 @@ nav .logo {
 }
 
 header {
-	line-height: 0.75;
-	margin: 50px 0px 50px 100px;
-	width: 876px;
+	margin: 50px auto;
+	width: 100%;
 	display: flex;
+	justify-content: center;
+	align-items: center;
 }
 
 header .title {
-	font-size: 68px;
+	font-size: 72px;
```

Comment:
research how em and rem units for font-sizes, margin, and padding can be useful in responsive design.